index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="Styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;500;700&display=swap" rel="stylesheet">
+    <link
+      href="//db.onlinewebfonts.com/c/23c0fcab84d99da0de762de7e220a6e1?family=Europa"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="http://fonts.cdnfonts.com/css/eurostile-extended"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  </head>
+  <body>
+    <nav>
+      <img src="./images/mf-logo.svg" alt="" class="brand-img" />
+
+      <div class="nav-right">
+        <span class="nav-text">Mentorship</span>
+        <button class="nav-button">Say Hello</button>
+      </div>
+    </nav>
+
+    <main>
+      <h1>Front-end Developer</h1>
+      <p class="main-p">
+        I design and code beautifully simple things, and I love what I do
+      </p>
+      <img src="./images/mf-avatar.svg" alt="Graphic-profile-pic" class="img-avatar" />
+      <br />
+      <img src="./images/hero.svg" alt="Coding process" class="img-machine" />
+    </main>
+    <section>
+      <div class="text">
+        <h2>Hi, I'm Rami. Nice to meet you</h2>
+        <p>
+          From when I was a teenager I was interested in Web development, especially 
+          the front side of it since I prefer visuals and instantly seeing the changes
+          that I am doing. I'm currently doing an internship that is teaching me new things
+          and allowing me to sharpen my skills. After it, I would like to graduate from Computer
+          Science and start my career.
+        </p>
+      </div>
+    </section>
+
+    <div class="card-info">
+      <div class="card-info-1">
+          <img src="./images/skills.svg" alt="">
+          <h1>Front-end Developer</h1>
```

Comment:
dont over use headers. and dont use a header number more than once. use p instead and change the font in css

Reply:
this shouldnt be the case
having only one header number only applies to h1
for all the lower numbers just be mindful if its a header use one of the h tags and use them in the proper sequence without skipping levels
only use p if its content